neqo-common/Cargo.toml

```diff
@@ -28,6 +28,8 @@ qlog = { workspace = true }
 windows = { version = "0.58", default-features = false, features = ["Win32_Media"] }
 
 [dev-dependencies]
+criterion = { version = "0.5", default-features = false }
+neqo-crypto = { path = "../neqo-crypto" }
```

Comment:
I think these should be regular dependencies gated by a `bench` feature, like we do in `neqo-transport/Cargo.toml`. (I'm surprised this compiled...)

Reply:
It's a dev-dependency, which can't be feature-gated.